package.json

```diff
@@ -17,7 +17,7 @@
     "format": "eslint . --fix",
     "babel": "babel --extensions '.js,.ts,.tsx' --no-babelrc --config-file=./babel.config.publish.js src --ignore '**/__tests__/**'  --copy-files --source-maps --delete-dir-on-start",
     "tsc": "tsc",
-    "build": "yarn babel --out-dir dist && del 'dist/**/__tests__' && yarn tsc --emitDeclarationOnly",
+    "build": "yarn babel --out-dir dist && rm -rf 'dist/**/__tests__' && yarn tsc --emitDeclarationOnly",
```

Comment:
Any reason for the change? `del` is cross-platform and has certain safeguards

Reply:
with `del`, I get "** is not a valid switch" -- I think in git bash it's execing in a cmd.exe command where `del` matches DOS & CP/M semantics and the glob isn't transformed. I never knew `del` to be cross platform.

Reply:
So according to the docs, changing `del` to `del-cli` should be enough

Reply:
done.